### PR TITLE
Self-documenting endpoint

### PR DESCRIPTION
**Pull request by reiinakano**
## Description

This is a very simple fix, I simply changed the endpoint to return the *docstring* of the selected `predict` function. Personally, I think this is the best way to keep it "self-documenting" and keeping the description in one place only. Let me know if you have problems with this approach.

Also, I added an `__init__.py` file to the `algorithms` folder to make it a valid Python package.
## Reference to official issue

#10 

## How Has This Been Tested?

Modified the `test_endpoint_documentation` test to make sure that calls return the appropriate docstring.

## CLA
- [x] I have signed the CLA; if other committers are in the commit history, they have signed the CLA as well

### CODE REVIEW

**Comment by reiinakano** [comment]
Travis test failed due to Git LFS issues probably.
```
warning: Clone succeeded, but checkout failed.

You can inspect what was checked out with 'git status'

and retry the checkout with 'git checkout -f HEAD'
```

File: prediction/src/tests/test_endpoints.py [hunk 1]
```diff
@@ -11,6 +11,9 @@
 
 from flask import url_for
 from src.factory import create_app
+from src.algorithms import classify
```
Comment (lamby) [inline prediction/src/tests/test_endpoints.py]: You can write this more cleanly as:

```
from src.algorithms import classify, identify, segment
```

File: prediction/src/tests/test_endpoints.py [hunk 2]
```diff
@@ -48,10 +51,22 @@ def test_endpoint_documentation(client):
...
+
+    url = client.url_for('predict', algorithm='classify')
+    r = client.get(url)
+    data = get_data(r)
+
+    assert data['description'] == classify.trained_model.predict.__doc__
 
 
-def test_indentify(client):
+def test_identify(client):
```
Comment (lamby) [inline prediction/src/tests/test_endpoints.py]: :D

File: prediction/src/tests/test_endpoints.py [hunk 3]
```diff
@@ -48,10 +51,22 @@ def test_endpoint_documentation(client):
...
-    assert 'description' in data
+    assert data['description'] == identify.trained_model.predict.__doc__
+
+    url = client.url_for('predict', algorithm='segment')
+    r = client.get(url)
+    data = get_data(r)
+
+    assert data['description'] == segment.trained_model.predict.__doc__
+
+    url = client.url_for('predict', algorithm='classify')
```
Comment (lamby) [inline prediction/src/tests/test_endpoints.py]: Feels like we could at least put this into a loop or a test helper instead of duplicating the code twice.

**Comment by reiinakano** [comment]
@lamby Fixed. But test failed again due to LFS issues

**Comment by lamby** [comment]
@reiinakano Thanks. Lets see how those tests do before merging, but please squash your commits so we don't have an ugly "fix tests" commit ;)

**Comment by reiinakano** [comment]
Hi @lamby, Github has a one click option to squash all commits when merging to master. It's what most open source projects do now. Wouldn't that be enough? This branch will be deleted after merging anyway. All that'll be left is a one nice commit on the master branch history.

**Pull request merged**